augur/mask.py

```diff
@@ -122,7 +134,10 @@ def mask_fasta(mask_sites, in_file, out_file):
 
 def register_arguments(parser):
     parser.add_argument('--sequences', '-s', required=True, help="sequences in VCF or FASTA format")
-    parser.add_argument('--mask', required=True, help="locations to be masked in BED file format")
+    parser.add_argument('--mask', dest="mask_file", required=False, help="locations to be masked in BED file format")
+    parser.add_argument('--mask-from-beginning', type=int, help="FASTA Only: Number of sites to mask from beginning")
+    parser.add_argument('--mask-from-end', type=int, help="FASTA Only: Number of sites to mask from end")

```

Comment:
These two arguments need to have default values of 0 or they will get passed as `None` to the downstream math in `mask_fasta` and produce this exception:

```
Removing masked sites from FASTA file.
Traceback (most recent call last):
  File "./bin/augur", line 24, in <module>
    exit( main() )
  File "./augur/__main__.py", line 10, in main
    return augur.run( argv[1:] )
  File "./augur/__init__.py", line 74, in run
    return args.__command__.run(args)
  File "./augur/mask.py", line 197, in run
    mask_from_end=args.mask_from_end)
  File "./augur/mask.py", line 121, in mask_fasta
    if beginning + end > sequence_length:
TypeError: unsupported operand type(s) for +: 'NoneType' and 'NoneType'
```